code/src/pages/products/ProductsPage.scss

```diff
@@ -0,0 +1,47 @@
+@import "@design-system";
+
+.spa-products-page {
+  padding: spacing(10);
+  background-color: var(--spa-foreground-background-color, $white);
+  display: flex;
+  flex-direction: column;
+  gap: spacing(5);
+
+  &__header {
+    font-weight: 600;

```

Comment:
Use defined value from design system

Reply:
> Use defined value from design system

added `semi-bold` with 600 and rewrited that

Reply:
It is not correct solution. `Lato` font does not have 600 font weight, you can check it on google fonts. I suggest using 700 font weight

Reply:
> It is not correct solution. `Lato` font does not have 600 font weight, you can check it on google fonts. I suggest using 700 font weight

removed that `variable` with value